static/src/stylesheets/module/email/_form.scss

```diff
@@ -3,10 +3,12 @@
 }
 
 .email-sub__iframe {
-    // pixel calculations are manual design 
-    // adjustments, off grid
-    width: calc(100% + 8px);
-    margin-left: -4px;
+    @if ($old-ie) {
+        width: 100%;
+    } @else {
+        width: calc(100% + 8px);
```

Comment:
@superfrank why can they not be part of the grid calcs? if we change the grid, will those still be the correct values?